kubernetes/operationcode_python_backend/overlays/staging/ingress.yaml

```diff
@@ -44,6 +45,14 @@ spec:
         backend:
           serviceName: resources-api-service
           servicePort: 80
+      - path: /metrics
```

Comment:
Looking at the comment about redirects coming first, should this come before the "/*"? 

Im guessing its applying the rules from top to bottom based on some of the examples [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#name-based-virtual-hosting). May be worth at least checking it with the other order if this doesnt work.

Reply:
Hmm, I'm not sure. I'm fine changing it just in case